src/index.js

```diff
@@ -104,12 +105,18 @@ export default {
     );
     return snapshotLayout;
   },
-  getContextMenu (obj, menu) {
+  async getContextMenu (obj, menu) {
     if (!this.$scope.layout.allowexportxls) {
       return menu;
     }
 
-    if (this.backendApi.model.layout.qMeta.privileges.indexOf('exportdata') !== -1) {
+    // Export as XLS is removed from desktop because the desktop wrapper blocks downloads.
+    // isPersonalMode returns true for both desktop and QCS
+    // By checking both if has download dialog and if is QCS can enable Export as XLS option on QCS
+    const app = qlik.currApp(this);
```

Comment:
It may work to use
```suggestion
    const app = this.backendApi.model.app;
```
and it will probably cause isPersonalResult to be a boolean instead of having to check isPersonalResult.qReturn

Reply:
I fix it as your comment said.
Tested on desktop, the option is not showing.
Tested on windows server, the option is showing and can download file.
Need to test on QCS when release a new version.